R/loadReg.R

```diff
@@ -208,10 +208,13 @@ resampleCoefficients.loadReg <- function(fit, flux.or.conc) {
 #'   the same model structure except for whether the left-hand side of the model
 #'   formula is flux or concentration. Some of the model metrics differ between 
 #'   these two internal models.
+#' @param constituent char the constituent being predicted
+#' @param site char Site ID water quality measurements
 #' @importFrom smwrStats rmse
 #' @importFrom stats coef
 #' @export
-summarizeModel.loadReg <- function(load.model, flux.or.conc=c("flux", "conc"), ...) {
+summarizeModel.loadReg <- function(load.model, flux.or.conc=c("flux", "conc"),
+                                   site = NULL, constituent = NULL, ...) {
```

Comment:
there's another approach that i'd prefer here if you can make it work: see https://github.com/USGS-R/loadflexBatch/pull/8#discussion_r98796000